docs/reference/licenses.nuget.org.md

```diff
@@ -0,0 +1,59 @@
+# licenses.nuget.org
+
+## Rationale
+
+With introduction of the [license expressions](nuspec.md#license) a requirement to have a reliable service that would
+provide a reference text for individual license identifiers, exception identifiers or license expression emerged.
+The one with a stable URL schema, that is not susceptible to the link rot, so that we can safely use it to provide
```

Comment:
The one?

Maybe `An additional requirement for this service is to have a stable...`